ci/test.lua

```diff
@@ -189,39 +189,67 @@ local function ensure_title_screen()
                          dfhack.gui.getCurFocus(true)[1]))
 end
 
-local function is_fortress(focus_string)
-    focus_string = focus_string or dfhack.gui.getCurFocus(true)
-    return focus_string == 'dwarfmode/Default'
+local function is_fortress()
+    return dfhack.gui.matchFocusString('dwarfmode/Default')
+end
+
+local function click_top_title_button(scr)
+    local sw, sh = dfhack.screen.getWindowSize()
+    df.global.gps.mouse_x = sw // 2
+    df.global.gps.precise_mouse_x = df.global.gps.mouse_x * df.global.gps.tile_pixel_x
+    if sh < 60 then
+        df.global.gps.mouse_y = 23
+    else
+        df.global.gps.mouse_y = (sh // 2) + 1
+    end
+    df.global.gps.precise_mouse_y = df.global.gps.mouse_y * df.global.gps.tile_pixel_y
+    df.global.enabler.tracking_on = 1
+    df.global.enabler.mouse_lbut = 1
+    df.global.enabler.mouse_lbut_down = 1
+    dfhack.screen._doSimulateInput(scr, {})
+end
+
+local function load_first_save(scr)
+    if #scr.savegame_header == 0 then
```

Comment:
To avoid unexpectedly altering a user fort, I think it would be good to check that there is exactly one save, with the expected name, and bail out if not.

Reply:
good idea; added